CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+## 1.0.0-beta.27
+
+### Bug Fixes
+
+Removes the items-center on megabuttons, fixing the alignment issue.
```

Comment:
Try to put yourself in the shoes of someone not on our team reading these entries.  Would you know what "the alignment issue" was?  Maybe this can be rephrased to something like:

> Fixes an issue with the alignment of the `text` in the `MegaButton` component so that when used in a group all of the text across `MegaButton` components is vertically aligned.